src/commands/config/set.ts

```diff
@@ -80,47 +101,82 @@ const attemptWriteToFile = (newConfig, value, dotNotation) => {
 	return result;
 };
 
-const setValue = (config, dotNotation, value) => {
+const setValue = (
+	config: ConfigOptions,
+	dotNotation: string,
+	value: WritableValue,
+) => {
 	setNestedConfigProperty(config, dotNotation, value);
+
 	return attemptWriteToFile(config, value, dotNotation);
 };
 
-const setBoolean = (config, dotNotation, value) => {
+const setBoolean = (
+	config: ConfigOptions,
+	dotNotation: string,
+	value: string,
+) => {
 	if (!checkBoolean(value)) {
 		throw new ValidationError('Value must be a boolean.');
 	}
 	const newValue = value === 'true';
+
 	return setValue(config, dotNotation, newValue);
 };
 
-const setArrayURL = (config, dotNotation, value, inputs) => {
+const setArrayURL = (
+	config: ConfigOptions,
+	dotNotation: string,
+	_: string,
```

Comment:
this argument is needed?

Reply:
yes, this one is not used but the next `inputs` is used